contents.json

```diff
@@ -24357,6 +24357,29 @@
       "date_added": "Feb 04 2021",
       "suggested_by": "@vinhnx",
       "stars": 12
+    },
+    {
+      "title": "Monotone",
+      "category-ids": [
+        "content",
+        "photo",
+        "rxswift",
+        "social"
+      ],
+      "tags": [
+        "swift"
+      ],
+      "description": "Modern Mobile Application for Exploring Unsplash.",
+      "source": "https://github.com/Neko3000/Monotone",
+      "screenshots": [
+        "https://raw.githubusercontent.com/Neko3000/resource-storage/master/projects/monotone/screens_en/screen-shot-1.png",
+        "https://raw.githubusercontent.com/Neko3000/resource-storage/master/projects/monotone/screens_en/screen-shot-2.png",
+        "https://raw.githubusercontent.com/Neko3000/resource-storage/master/projects/monotone/screens_en/screen-shot-3.png",
+        "https://raw.githubusercontent.com/Neko3000/resource-storage/master/projects/monotone/screens_en/screen-shot-4.png",
```

Comment:
hi @Neko3000 thanks for the contribution, could you remove the extra comma at the end of this line?